contracts/utils/structs/DoubleEndedQueue.sol

```diff
@@ -152,17 +155,16 @@ library DoubleEndedQueue {
      * @dev Returns the number of items in the queue.
      */
     function length(Bytes32Deque storage deque) internal view returns (uint256) {
-        // The interface preserves the invariant that begin <= end so we assume this will not overflow.
-        // We also assume there are at most int256.max items in the queue.
+        // We also assume there are at most uint128.max items in the queue.
```

Comment:
I don't think this code is assuming anything so the comment can be removed.

Reply:
```suggestion
```

Reply:
done